Initial Model - CNN.ipynb

```diff
@@ -0,0 +1,785 @@
+{
```

Comment:
<p>could you add these libraries to <code>requirements.txt</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/akaashsidhu/SM_project/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>